psi4/src/psi4/libmints/writer.cc

```diff
@@ -46,6 +46,12 @@
 using namespace psi;
 ;
 
+
+MoldenWriter::MoldenWriter(std::shared_ptr<Wavefunction> wavefunction) : wavefunction_(wavefunction) {
```

Comment:
Why did you duplicate the function from line 55. The compiler is reporting an error.  This should have come up on your local compile. One of them must be deleted.